src/assets/styles/main.scss

```diff
@@ -14,8 +14,10 @@
 @import "app";
 // Pages
 @import "pages/homepage";
-@import "pages/boardspage";
+@import "pages/workspacepage";
```

Comment:
you removed the pages/boardspage.

pay attention when you resolve conflicts. do not remove code from another developers. please rollback